resources/views/layouts/navigation.blade.php

```diff
@@ -25,6 +26,9 @@ class="menu menu-sm dropdown-content mt-3 z-[1] p-2 shadow bg-gray-200 rounded-b
                     </button>
                     </li>
                     </ul>
+                    @else
+                        <a href="{{ route('register') }}">Sign in</a>
```

Comment:
Maybe it would be better to route them into the login page? Or change the "Sign in" to "register". From the standpoint of the average user, it might appear confusing

Reply:
Agreed, I think "Sign in" should be changed to "Register" to more accurately reflect the route.